documentation/src/docs/beginner_dev_guide.md

```diff
@@ -0,0 +1,200 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega - Creating Your First Application
+
+Learn how to create a Hello World Pravega app. This guide covers:
+
+* Starting Pravega standalone.
+* Creating a Pravega Stream.
+* Creating an Event Writer and write events into a Pravega Stream.
+* Create an Event Reader and read events from the Pravega Stream.
+
+ 
+# 1. Prerequisites
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 11+ installed with JAVA_HOME configured appropriately
+* Gradle 6.5.1+
+
+<details open>
+  <summary>Gradle 6.5.1+</summary>
+  Installation : https://gradle.org/install/
+<p>
+  !! Verify Gradle is using the Java you expect. You can verify which JDK Gradle uses by running `gradle --version`. !!
+</details>
+
+# 2. Goal
+In this guide, we will create a straightforward application that creates a Stream on Pravega and writes an event into the Stream and reads back from it.
+We recommend that you follow the instructions from [Bootstrapping project](#4-Bootstrapping-the-Project) onwards to create the application step by step.
+However, you can go straight to the completed example at [Pravega-samples-repo](https://github.com/pravega/pravega-samples).
+
+<details open>
```

Comment:
I don't think that using this `<details option>` at each instruction is desirable, as this is something that we expect users to see (not an option), right?

Reply:
To me, new user guides are about revealing the commands needed to walk through various steps... Without the `open` attribute there, readers have to click an expander tab to see the commands that the doc is supposed to be all about.